bootstrap.sh

```diff
@@ -1 +1,21 @@
 #!/bin/bash
+
+kubectl apply -f .infrastructure/namespace.yml
+kubectl create namespace mysql
+
+# Apply MySQL StatefulSet and ConfigMap
+kubectl apply -f .infrastructure/mysql/mysql-secret.yml
+kubectl apply -f .infrastructure/mysql/init-configMap.yml
+kubectl apply -f .infrastructure/mysql/mysql-headless-service.yml
+kubectl apply -f .infrastructure/mysql/statefulSet.yml
+
+#Create todoaapp and services
```

Comment:
Typographical error in the comment: 'todoaapp' should be corrected to 'todoapp'.